Mods/vcmi/config/vcmi/rmg/hdmod/frozenDragons.JSON

```diff
@@ -12,7 +12,7 @@
 				"owner" : 1,
 				"monsters" : "normal",
 				"playerTowns" : { "castles" : 1 },
-				"allowedTowns" : [ "rampart", "tower", "inferno", "dungeon", "stronghold", "fortress" ],
+				"bannedTowns" : ["castle", "necropolis", "conflux"],
 				"allowedMonsters" : [ "castle", "tower", "inferno", "stronghold", "fortress", "neutral", "conflux" ],

```

Comment:
How do we handle checks like this when considering mods?
Will this check also ban all creatures from mods?

In that case we might want to invert it to "bannedMonsters"?

Reply:
Yes, it will not allow creatures from new factions. However, I don't think it has much impact on balance. It's more for a look & feel, or to tell the zones apart.
Surely I could add "banned Monsters", and even  "banned terrains" as well.

Reply:
Understood.

Well, up to you. No idea whether this option is actually needed.